docs/contents/index.md

```diff
@@ -60,20 +60,21 @@ and the schema can be found in the
 [eks-distro-build-tooling](https://github.com/aws/eks-distro-build-tooling/tree/main/release)
 GitHub repository.
 
-You can install the CRD API type, the release channel for Kubernetes 1-18, and
-view the release channel by running the following commands:
+You can install the CRD API type, the release channel for EKS-D, and view the release channel
```

Comment:
Should we call it `release channel manifest` here in alignment with the `release manifest` terminology in line 102?

Reply:
Yeh, I like that idea. Done.